pynws/nws.py

```diff
@@ -49,7 +49,8 @@ async def get_stations_observations(self, limit=0, start_time=None):
         res = await raw_stations_observations(
             self.station, self.session, self.userid, limit, start_time
         )
-        return [o["properties"] for o in res["features"]]
+        observations = [o["properties"] for o in res["features"]]
+        return sorted(observations, key=lambda o: o.get("timestamp"))
```

Comment:
I think the default is to be in reverse sorted order?

Reply:
That might be. It's hard to tell right now when they're out of order.

Reply:
I was looking for the most recent observation--which is likely the most common use case.

I found that there's a specific URL for that: /stations/{stationId}/observations/latest

Reply:
> stations_observations_multiple.json test data has them sorted in ascending order.

This data actually seems to be malformed for "timestamp".  The value is the same for both entries.  The other "multiple" is "unsorted" meaning it is out of order to the default NWS style.  I think the default is descending, i.e. newest first.

You can use `limit=1` to get only the last observation in the current code.  Do you think the other end point has utility beyond this usage?

Reply:
I tried using limit=1 but it didn't get the latest entry.

Reply:
Ah this is a really bad bug on the NWS servers then.  I don't mind adding it to this library.

Reply:
> I think the default is to be in reverse sorted order?

Any suggestions on how to answer this definitively while their API is broken?

Reply:
I can't answer definitely with a source, but an additional pointer is that this line selects the first valid observation.  This requires the observations to be sorted in descending order.  This worked with the default NWS response prior to #60.

https://github.com/MatthewFlamm/pynws/blob/7428f0a01e532c770d98889e6a64af1e8e7cf3a0/pynws/simple_nws.py#L271